src/KeyVault/KeyVault/Commands/NewAzureKeyVault.cs

```diff
@@ -84,6 +87,9 @@ public class NewAzureKeyVault : KeyVaultManagementCmdletBase
             HelpMessage = "If specified, enables secrets to be retrieved from this key vault by Azure Disk Encryption.")]
         public SwitchParameter EnabledForDiskEncryption { get; set; }
 
+        public const String DisableSoftDeleteChangeDesc = "DisableSoftDelete will be deprecated without being replaced by December 2020";
+
+        [CmdletParameterBreakingChange("DisableSoftDelete", ChangeDescription = DisableSoftDeleteChangeDesc)]
```

Comment:
We can also use version 3.0.0 here, because those properties will be deprecated in October not December.